python/ray/tune/function_runner.py

```diff
@@ -304,14 +308,16 @@ def save(self, checkpoint_path=None):
             checkpoint, parent_dir, state)
         return checkpoint_path
 
-    def save_to_object(self):
+    def save_to_object(self, remove=False):
```

Comment:
As far as I can see, this function is currently never invoked with `remove=True`. Is this correct or is this missing somewhere?